docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/FeatureSelection/SelectFeaturesBasedOnCount.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class SelectFeaturesBasedOnCount
+    {
+        private static readonly int printRowCount = 4;
+
+        public static void Example()
+        {
+            // Downloading a classification dataset from github.com/dotnet/machinelearning.
+            // It will be stored in the same path as the executable
+            string dataFilePath = SamplesUtils.DatasetUtils.DownloadBreastCancerDataset();
+
+            // Data Preview
+            //    1. Label							0=benign, 1=malignant
```

Comment:
>							 [](start = 26, length = 7)

He use tabs! Where is my pitchfork!

Reply:
:) lol

---
In reply to: [271907898](https://github.com/dotnet/machinelearning/pull/3184#discussion_r271907898) [](ancestors = 271907898)